src/linagora.esn.unifiedinbox/app/components/email/footer/email-footer.pug

```diff
@@ -1,16 +1,16 @@
 .lv-footer
-  .action
-    a(ng-click="ctrl.forward()", title="{{ 'Forward' | translate }}")
-      i.mdi.mdi-forward
-      span.label
-        | {{ 'Forward' | translate }}
   .action
     a(ng-click="ctrl.reply()", title="{{ 'Reply' | translate }}")
       i.mdi.mdi-reply
       span.label
         | {{ 'Reply' | translate }}
-  .action(ng-if="::email.hasReplyAll")
+  .action(ng-if="::ctrl.email.hasReplyAll")
```

Comment:
Do you need to add controller here ?

Reply:
It's a fix, currently you don't see the reply all because we forget that during a refacto ;)